docs/developer/decision-records/2024-08-23-identity_hub_identity_write_credentials_api/README.md

```diff
@@ -0,0 +1,16 @@
+# Identity Hub - Identity Write Credentials API
+
+## Decision
+
+Write endpoints (create/update) will be added to the Identity Hub identity API.
+
+## Rationale
+
+As of now the issuance of VCs into the Identity Hub is not defined as part of the DCP specification. Thus, it is
```

Comment:
TBH, this rationale is dangerous in that it insinuates that this API is a replacement for issuance, which **it is not**.

IMO, the reasoning should be "to give users a way to author VCs, to add VCs that weren't issued using DCP Issuance, but using some other means, or enable users to populate the VC storage with existing VCs after moving to a new IH instance or similar"

In fact, I would state explicitly, that this API is **not** to be used for VC issuance directly.